lib/ansiblelint/rules/__init__.py

```diff
@@ -0,0 +1 @@
+"""Hosts all internal ansible-lint rules."""
```

Comment:
This is a package docstring, not a function docstring. For "objects" we describe what they are. For functions/methods we describe what they *do*. So use a verb only for those.
```suggestion
"""All internal ansible-lint rules."""
```